gateway/src/main/java/com/mx/path/gateway/configuration/AccessorStackConfigurator.java

```diff
@@ -116,6 +119,8 @@ public Accessor buildAccessor(String name, ObjectMap map, String clientId, Objec
         }
       }
 
+      invokeAfterInitializeMethods(accessor.getClass());
```

Comment:
At this point, `accessor` is actually a proxy. The proxy will never have this method. If you move this down to `buildFromNode`, you will have the original accessor class and this should work.

Reply:
This is on line 122, way below than the `buildFromNode` method call. Do you want me to move `invokeAfterInitializedMethods` into the `buildFromNode` function?

Reply:
I'm not sure what you're asking. Are you asking where to put the invokeAfterInitializedMethods function or where to call it?

Reply:
Where to call it, I does matter where I put the function.

Reply:
Yes. I'm saying we could move that call into buildFromNode. It needs to be in there so it has access to the raw accessor before it is wrapped in the proxy.

Reply:
Done.